src/API.js

```diff
@@ -1,48 +1,19 @@
-import axios from "axios"
+import axios from 'axios'
 
 const API = {
-    SERWER: "http://localhost:3000",
-    TOKEN: "0123456789",
+  baseHeaders: {
+    headers: {
+      Authorization: process.env.REACT_APP_TOKEN
+    }
```

Comment:
get it from local storage then and check for the presence before you actually execute the call